codex/utils/asyncheapqueue.nim

```diff
@@ -282,7 +282,7 @@ proc len*[T](heap: AsyncHeapQueue[T]): int {.inline.} =
 
 proc size*[T](heap: AsyncHeapQueue[T]): int {.inline.} =
   ## Return the maximum number of elements in ``heap``.
-  len(heap.maxsize)
+  heap.maxsize
```

Comment:
The compiler does not like `len(heap.maxsize)`, because there is no implementation for `len*(x: int)`. I'm assuming this was not caught before due to it never being consumed.

The `len` proc above is correct, because it ultimately calls `len*[T](x: seq[T])`, of which there is an implementation for in the standard library.